### ISSUE TITLE
Network Loading Control API (hls.js) equivalent for Shaka Player

### ISSUE THREAD

**Issue by radiantmediaplayer** [shaka-project/shaka-player#4851]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Y

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Y

**What version of Shaka Player are you using?**
4.3.2

**Please ask your question**
Is there an equivalent to the Network Loading Control API found in hls.js with Shaka Player, see https://github.com/video-dev/hls.js/blob/master/docs/API.md#network-loading-control-api.
Basically we would like to implement a stop download on pause feature. Is there already an equivalent for this in Shaka Player? If yes what is the method that needs to be called?
Thanks

**Comment by joeyparrish** [comment]
We don't have something exactly like that.  Instead, our loading is governed by the buffering configs:  https://shaka-player-demo.appspot.com/docs/api/tutorial-network-and-buffering-config.html

When `bufferingGoal` is met, we stop loading segments.  If it's not met, we load another segment.

So if you pause playback, we will continue buffering until the `bufferingGoal` setting is satisfied, then stop automatically.

Does this answer your question?

**Comment by radiantmediaplayer** [comment]
Thanks for your answer.We will work with this for the time being.
